lib/src/mocks/mock_http.dart

```diff
@@ -32,9 +32,9 @@ class MockHttp {
   }
 
   @Deprecated(v3Deprecation)
-  void completeRequest(BaseRequest? request, {BaseResponse? response}) {
+  void completeRequest(BaseRequest request, {BaseResponse? response}) {
```

Comment:
completeRequest is not exported from w_transport, so we can be a bit more bold with the changes here

There was only a single location where `completeRequest` was passing a null in, it was within a unit test that was verifying it could handle a null value

This can be improved by just updating `completeRequest` to not accept a non-nullable value